tools/contrib/neon/neon_runner.py

```diff
@@ -66,19 +65,17 @@
 import argparse
 import subprocess
 import pandas as pd
- 
+import glob 
 from datetime import date
 from getpass import getuser
  
-#-- get the environment variable
-cesmroot = os.environ.get('CESM_ROOT')
+# Get the ctsm util tools and then the cime tools.
+_CTSM_PYTHON = os.path.abspath(os.path.join(os.path.dirname(__file__), "..","..","..",'python'))
+print("{}".format(_CTSM_PYTHON))
+sys.path.insert(1, _CTSM_PYTHON)
```

Comment:
I really like this much better than using the CESM_ROOT  environment variable. 👍